tool/tctl/common/resource_command.go

```diff
@@ -729,15 +792,16 @@ func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI,
 		return trace.Wrap(err)
 	}
 	exists := (err == nil)
-	if !rc.IsForced() && exists {
-		return trace.AlreadyExists("node %q with Hostname %q and Addr %q already exists, use --force flag to override",
+	if !rc.force && exists {
```

Comment:
We should either use `rc.force` everywhere and remove `IsForced` or use `IsForced` consistently.

Reply:
I believe that was done in https://github.com/gravitational/teleport/pull/30473/commits/bf40802d45a16116b346deb2daebfee5e2ce8b0e#diff-0ded7335cb699ba03c82523badbb1b4dd5db4e605d34b89b26b88e3942e1f94dL1469-L1473

Reply:
Nice